tools/docker/syzbot/Dockerfile

```diff
@@ -30,24 +30,15 @@ RUN curl https://dl.google.com/go/go1.22.7.linux-$(uname -m | sed 's/aarch64/arm
 ENV PATH /usr/local/go/bin:$PATH
 
 # The default clang-14 is too old, install the latest one.
+RUN curl -L https://mirrors.edge.kernel.org/pub/tools/llvm/files/llvm-15.0.7-x86_64.tar.gz | tar -xzf - --strip-components=2 -C /usr/bin llvm-15.0.7-x86_64/bin
```

Comment:
These files are signed, do you have an idea on how we can verify those signatures?

Reply:
Indeed. I've found a file with the [sha256 checksums](https://mirrors.edge.kernel.org/pub/tools/llvm/files/sha256sums.asc) for the integrity, for the signature (https://mirrors.edge.kernel.org/pub/tools/llvm/files/llvm-15.0.7-x86_64.tar.sign) let me check the public PGP key.
Will update the PR with the sha256 check and gpg signature verification.

Reply:
@ramosian-glider PTAL. I've moved the archive download step and all integrity & signature checks in separate bash script